generators/client/templates/vue/src/main/webapp/app/shared/data/data-utils.service.ts.ejs

```diff
@@ -78,10 +78,10 @@ export default class JhiDataUtils extends Vue {
    */
   clearInputImage(entity, elementRef, field, fieldContentType, idInput) {
     if (entity && field && fieldContentType) {
-      if (entity.hasOwnProperty(field)) {
+      if (Object.prototype.hasOwnProperty.call(entity, field)) {
         entity[field] = null;
       }
-      if (entity.hasOwnProperty(fieldContentType)) {
+      if (Object.prototype.hasOwnProperty.call(entity, fieldContentType)) {
```

Comment:
These are errors from es lint, which are now fixed/

Reply:
could we use the following script, and remove the 'if' condition?
`Object.keys(entity).forEach`